### ISSUE TITLE
[ERROR]:  The Recent FNF Update made my ported mods dont work

### ISSUE THREAD

**Issue by MNHBAS** [FNF-Porter/Porter.py#59]
### What is the preset selected?

Full mod

### If you chose what to convert, what did you pick? (if applicable)

The recent update (the pit shop update, where they added 2 erect songs and revamped the freeplay and results screen) made all my mods before that update not work, every time I go to freeplay, the game closes instantly

### What platform?

None

### Issue Checklist

- [X] I have properly named the issue
- [ ] I looked in issues/discussions, if my question was previously anwsered

**Comment by IamtheScouthere2006** [comment]
At lest I am not the only one having this problem, the only thing you can do now is play the old build until the FNF porter updates sadly.

**Comment by BombasticTom** [comment]
I'll have to look into it once I get free time

**Comment by Willio2000** [comment]
yep i had problems too, it seems like it happens with songs that cant load their stage but im not entirely sure
